OrbitClientServices/include/OrbitClientServices/TracepointServiceClient.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_TRACEPOINTSERVICECLIENT_H
+#define ORBIT_TRACEPOINTSERVICECLIENT_H
+
+#include <memory>
+#include <vector>
+
+#include "OrbitBase/Result.h"
+#include "grpcpp/channel.h"
+#include "services.grpc.pb.h"
+#include "tracepoint.pb.h"
+
+using orbit_grpc_protos::TracepointInfo;
+
+class TracepointServiceClient {
+ public:
+  TracepointServiceClient(const std::shared_ptr<grpc::Channel>& channel);
```

Comment:
Make this ctor private as well. The only way to get a TracepointServiceClient should be the static factory function "Create" .

Reply:
I tried to do this yesterday but it caused some compilation errors due to making a unique pointer to a TracepointServiceClient  instance. I did not fully understand this error because I thought that since the constructor is private and it is accessed only in that class's scope, it should be fine. However, I will investigate this further and try to find out why this happened.
![image](https://user-images.githubusercontent.com/52468725/91304122-a879be80-e7b1-11ea-9d97-2c41f97c3ffa.png)

Reply:
Just calling "new" in the Create function and handing the raw pointer over to a unique_ptr should do.